library/src/main/java/ai/promoted/PromotedAi.kt

```diff
@@ -88,9 +88,16 @@ object PromotedAi {
     /**
      * @see [PromotedAiSdk.onViewVisible]
      */
+    @Deprecated("View logging is now automatic")
```

Comment:
I think we're not quite at deprecating this yet. Our recommended integration will be to use auto view logging but for some use cases the manual `logView` can still be necessary. For example, if there's a few that doesn't trigger any impressions or actions.